pkgs/tools/misc/usbmuxd2/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, clangStdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, libimobiledevice
+, libusb1
+, avahi
+, clang
+}: let
+
+  libgeneral = clangStdenv.mkDerivation rec {
```

Comment:
Why not move this to top-level?

Reply:
This library is only used by projects of tihmstar. No other project is packaged yet here so for now I wouldn't package it at top-level